app/controllers/patients_controller.rb

```diff
@@ -0,0 +1,30 @@
+class PatientsController < Clearance::UsersController
```

Comment:
Forgot to mention that we won't need a controller for patients. Please change this to `admins`. We'd only keep a model for patients